controls/radexpander/features/header-button-style.md

```diff
@@ -0,0 +1,72 @@
+                <Setter Property="Foreground" Value="Red"/>
+            </Style>
+        </telerik:RadExpander.HeaderButtonStyle>
+    </telerik:RadExpander>
+{{endregion}}
+
+__RadExpander with a custom HeaderButtonStyle__
+
+![](images/radexpander-features-header-button-style-0.png)
+
+## Retain the Default RadToggleButton Look
+
+To keep the default look of the RadToggleButton while applying the needed customizations, you can base the custom style on its default one. A different method can be taken depending on the chosen theming approach.
+
+If the [__Implicit Styles__](https://docs.telerik.com/devtools/wpf/styling-and-appearance/styling-apperance-setting-a-theme-overview#setting-a-theme-using-implicit-styles) theming approach is used, you can base the custom style on the `ExpanderHeaderButtonStyle` style.
+
+#### __[XAML] Approach when applying a theme via Implicit Styles__
+{{region radexpander-features-header-button-style-1}}
+    <telerik:RadExpander Header="My Expander">
+        <telerik:RadExpander.HeaderButtonStyle>
+            <Style TargetType="telerik:RadToggleButton" BasedOn="{StaticResource ExpanderHeaderButtonStyle}">
+                <Setter Property="Foreground" Value="Red"/>
+            </Style>
+        </telerik:RadExpander.HeaderButtonStyle>
+    </telerik:RadExpander>
+{{endregion}}
+
+If the [__StyleManager__](https://docs.telerik.com/devtools/wpf/styling-and-appearance/styling-apperance-setting-a-theme-overview#setting-a-theme-using-stylemanager) theming approach is used, merge the resource dictionary for the __Telerik.Windows.Controls__ assembly.
+
+>important The __GenericFluent.xaml__ ResourceDictionary for the __Telerik.Windows.Controls__ assembly is generated by StyleManager for the `Fluent` theme. If another theme is used, for example, `Office_Black`, change the __GenericFluent.xaml__ to __GenericOfficeBlack.xaml__.
+
+#### __[XAML] Approach when applying a theme via StyleManager__
+{{region radexpander-features-header-button-style-2}}
+    <telerik:RadExpander Header="My Expander">
+        <telerik:RadExpander.Resources>
+            <ResourceDictionary>
+                <ResourceDictionary.MergedDictionaries>
+                    <ResourceDictionary Source="/Telerik.Windows.Controls;component/Themes/GenericFluent.xaml"/>
+                </ResourceDictionary.MergedDictionaries>
+            </ResourceDictionary>
+        </telerik:RadExpander.Resources>
+        <telerik:RadExpander.HeaderButtonStyle>
+            <Style TargetType="telerik:RadToggleButton" BasedOn="{StaticResource ExpanderHeaderButtonStyle}">
+                <Setter Property="Foreground" Value="Red"/>
+            </Style>
+        </telerik:RadExpander.HeaderButtonStyle>
+    </telerik:RadExpander>
+{{endregion}}
+
+Both of these approaches produce the following result:
+
+![RadExpander with custom Style based on its default one](images/radexpander-features-header-button-style-1.png)
```

Comment:
Why the button is aligned top-left. I would suggest to adjust the setup so that the button is centered vertically and it has a small margin on the left.